docs/reference-docs/system-tasks/json-jq-transform-task.md

```diff
@@ -5,7 +5,7 @@ sidebar_position: 1
 # JSON JQ Transform Task
 
 ```json
-"type" : "JSON_JQ_TRANSFORM_TASK"
+"type" : "JSON_JQ_TRANSFORM"
 ```
 ### Introduction
 
```

Comment:
@dougsillars in line 12 there's also a mention to `JSON_JQ_TRANSFORM_TASK` which should be just `JSON_JQ_TRANSFORM` because it might be misleading and cause the same issue @leandrocasadei experienced.